recipe/meta.yaml

```
@@ -27,13 +27,13 @@ requirements:
   run:
     - cinnabar ~=0.4.0
     - click
-    - gufe >=1.1.0,<1.2
+    - gufe ~=1.2
```

I think this will allow gufe 1.3 which I think we don't want
```suggestion
    - gufe ~=1.2.0
```

_should_ pull in a version like `1.2.1` but not `1.3.0`

---

ah perfect, that's what I wanted.

---

docs here: https://docs.conda.io/projects/conda-build/en/stable/resources/package-spec.html